RELEASE_NOTES.md

```diff
@@ -1,15 +1,16 @@
-# Cosmos SDK v0.47.4 Release Notes
+# Cosmos SDK v0.47.5 Release Notes
 
 💬 [**Release Discussion**](https://github.com/orgs/cosmos/discussions/categories/announcements)
 
 ## 🚀 Highlights
 
-Missed the v0.47.0 announcement? Read it [here](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.47.0).
-For this fourth patch release of the `v0.47.x` line, some of the notable changes include:
+Get ready for v0.50.0 and start integrating with the next [Cosmos SDK](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.50.0-rc.0) release.
 
-* An improvement in `<appd> prune` UX.
-* Improving the error handling when there is a snapshot creation failure.
+For this 5th patch release of the `v0.47.x` line, some of the notable changes include:
 
-Check out the [changelog](https://github.com/cosmos/cosmos-sdk/blob/v0.47.4/CHANGELOG.md) for an exhaustive list of changes or [compare changes](https://github.com/cosmos/cosmos-sdk/compare/release/v0.47.3...v0.47.4) from last release.
+* <!-- TODO -->
```

Comment:
I want to put this one https://github.com/cosmos/cosmos-sdk/pull/17274. Just waiting until it gets merged and backported.